swarm/storage/pyramid.go

```diff
@@ -112,41 +158,56 @@ func NewTreeEntry(pyramid *PyramidChunker) *TreeEntry {
 
 // Used by the hash processor to create a data/tree chunk and send to storage
 type chunkJob struct {
-	key       Key
-	chunk     []byte
-	size      int64
-	parentWg  *sync.WaitGroup
-	chunkType int // used to identify the tree related chunks for debugging
-	chunkLvl  int // leaf-1 is level 0 and goes upwards until it reaches root
+	key      Key
+	chunk    []byte
+	parentWg *sync.WaitGroup
```

Comment:
I don't understand why this is in `pyramid.go`. A `chunkJob` is chunker agnostic, isn't it?

Reply:
@nolash No, only the pyramid chunker uses the chunkJob type, it is an implementation specific private thing.